src/OpenTelemetry.Instrumentation.SqlClient/TracerProviderBuilderExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static class TracerProviderBuilderExtensions
             configureSqlClientInstrumentationOptions?.Invoke(sqlOptions);
 
             builder.AddInstrumentation(() => new SqlClientInstrumentation(sqlOptions));
-            builder.AddSource(SqlClientDiagnosticListener.ActivitySourceName);
+            builder.AddSource(new Source(SqlClientDiagnosticListener.ActivitySourceName));
```

Comment:
in the instrumentation example, should we use minversion and maxversion = the assembly version or should ignore our assembly version?

Reply:
just to update, we will leave blank for now.